packages/grid-pro/test/edit-column-renderer.test.js

```diff
@@ -198,7 +199,7 @@ describe('edit column renderer', () => {
       expect(spy.calledOnce).to.be.true;
     });
 
-    it('should exit the edit mode on custom editor component focusout event', () => {
+    it('should exit the edit mode on custom editor component focusout event', async () => {
```

Comment:
This `async` looks not needed anymore.

Reply:
Removed. Also had to change the event in `sendMouse` to `click` to reflect the latest changes in the `ClearButtonMixin`.